src/main/java/io/sprucehill/zalando/api/service/ICategoryService.java

```diff
@@ -0,0 +1,15 @@
+package io.sprucehill.zalando.api.service;
+
+import java.util.List;
+
+import io.sprucehill.zalando.api.exception.NotFoundException;
+import io.sprucehill.zalando.api.model.Category;
+import io.sprucehill.zalando.api.model.Domain;
+
+public interface ICategoryService {
+	
+	List<Category> list(Domain domain, String targetGroup,String[] keys) throws NotFoundException;
```

Comment:
There is no such method available at with the Zalando REST API; the Zalando REST API only offers listing all categories or getting a single category; please provide at least methods for all Zalando REST API endpoints;

Furthermore it seems that this method is not a listing, but a multi get instead; please use naming accordingly